8th commit:

waspp01 opened the pull request:
- Added ShoppingCart Entity, Service, Repository, DTOs , Controller, Mapper
- Added CartItem entity, DTOs, Service, Repository, Mapper
- Added ShoppingCart creation to UserServiceImpl
- Added required liquibase changes

waspp01 committed:
8th commit:
- Added ShoppingCart Entity, Service, Repository, DTOs , Controller, Mapper
- Added CartItem entity, DTOs, Service, Repository, Mapper
- Added ShoppingCart creation to UserServiceImpl
- Added required liquibase changes

koval-kn commented on src/main/java/com/example/onlinebookstore/controller/ShoppingCartController.java:
```diff
@@ -0,0 +1,73 @@
+
+@Tag(name = "Shopping cart management", description = "Endpoints for managing shopping cart")
+@RestController
+@RequestMapping(value = "/cart")
```
```suggestion
@RequestMapping("/cart")
```

koval-kn commented on src/main/java/com/example/onlinebookstore/service/impl/ShoppingCartServiceImpl.java:
```diff
@@ -0,0 +1,72 @@
+    public ShoppingCartDto addCartItem(Long userId,
+                                   CreateCartItemRequestDto requestDto) {
+        ShoppingCart shoppingCart = getShoppingCartById(userId);
+        List<CartItem> cartItems = shoppingCart.getCartItems();
```
consider extracting this check to the separate method

koval-kn reviewed (changes requested):
consider to improve your commit naming in future

waspp01 commented:
> consider to improve your commit naming in future

What do you mean? Should I add some info about implemented things instead of just number of commit?

waspp01 commented:
> consider to improve your commit naming in future

What do you mean? Should I add some info about implemented things instead of just number of commit?

waspp01 committed:
8th commit:
- Added ShoppingCart Entity, Service, Repository, DTOs , Controller, Mapper
- Added CartItem entity, DTOs, Service, Repository, Mapper
- Added ShoppingCart creation to UserServiceImpl
- Added required liquibase changes

koval-kn commented:
> > consider to improve your commit naming in future
> 
> What do you mean? Should I add some info about implemented things instead of just number of commit?

yeah :)
